packages/create-cli/src/utils/__tests__/directory.spec.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
I think that this is some compiled TS that could be deleted as well

Reply:
yep, seems like in the last 3 hours my IDE started storing these buggers.